eng/helix/content/InstallDotNet.ps1

```diff
@@ -28,8 +28,8 @@ param(
 
 & $PSScriptRoot\Download.ps1 "https://dot.net/v1/dotnet-install.ps1" $PSScriptRoot\dotnet-install.ps1
 Write-Host "Download of dotnet-install.ps1 complete..."
-Write-Host "Installing SDK...& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Version $sdkVersion -InstallDir $installDir"
-Invoke-Expression "& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Version $sdkVersion -InstallDir $installDir"
-Write-Host "Installing Runtime...& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Runtime dotnet -Version $runtimeVersion -InstallDir $installDir"
-Invoke-Expression "& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Runtime dotnet -Version $runtimeVersion -InstallDir $installDir"
-Write-Host "InstallDotNet.ps1 complete..." 
\ No newline at end of file
+Write-Host "Installing SDK...& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Version $sdkVersion -InstallDir $installDir -NoPath"
+Invoke-Expression "& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Version $sdkVersion -InstallDir $installDir -NoPath"
+Write-Host "Installing Runtime...& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Runtime dotnet -Version $runtimeVersion -InstallDir $installDir -NoPath"
+Invoke-Expression "& $PSScriptRoot\dotnet-install.ps1 -Architecture $arch -Runtime dotnet -Version $runtimeVersion -InstallDir $installDir -NoPath"
+Write-Host "InstallDotNet.ps1 complete..." 
```

Comment:
Why does this change show up here❔

BTW @HaoK you'll need to port #24968 to release/5.0. It missed today's branching.

Reply:
Okay I can file another PR unless we want it to go in with this one?

Reply:
For tracking, better separate but it's up to @pranavkm if validation passes this time

Reply:
Sorry missed this. I changed the branch after I created it, let me rebase on 5.0 since this build is busted anyway